etp-backend/src/main/clj/solita/etp/service/laskutus.clj

```diff
@@ -377,9 +391,8 @@
                                        laskutustieto-xml-files
                                        laskutustieto-destination-dir)
               (log/info "Laskutustieto xmls uploaded with SFTP."))
-
-            ;; TODO send täsmätytysraportti with SMTP.
-            )
+            (send-tasmaytysraportti-email! tasmaytysraportti-file)
+            (log/info "Täsmätysraportti sent as an email"))
         (log/warn "SFTP configuration missing. Skipping actual integration."))
       (delete-files! all-files)
```

Comment:
It is not changed in this change set, but I wonder if this delete should be in the `finally` section.

Reply:
That's true. Will fix.